docs/cmake.md

```diff
@@ -0,0 +1,33 @@
+# CMake
+
+> There are only two kinds of build systems: the ones people complain about and the ones nobody uses.  
+> ---  [lt_algorithm_gt](https://www.reddit.com/r/cpp/comments/ihi37f/what_is_the_point_of_this_talk_just_making_fun_of/g34ex3x?utm_source=share&utm_medium=web2x&context=3)
+> (adapted from Bjarne Stroustrup)
+
+Note, that building tests and benchmarks for __Skipper__ is enabled by default 
+(and for that, `Catch2` and `Google/Benchmark` are required).  
+In order to turn them off, set corresponding variables to `OFF`:
```

Comment:
Add path to that CMake file for more clarity.

Reply:
It's not necessary for projects that use `skipper` as a library. They should write this in their own `CMakeLists.txt`, not in ours.